codegen/src/api/mod.rs

```diff
@@ -115,13 +131,14 @@ pub fn generate_runtime_api_from_bytes(
     item_mod: syn::ItemMod,
     bytes: &[u8],
     derives: DerivesRegistry,
+    type_substitutes: TypeSubstitutes,
```

Comment:
Super small nit: missing 
```/// * `type_substitutes` - Provide custom type substitutes.```
 on the doc.

Tho, I'm wondering if we should extend this documentation style to our repo, or just remove those `# Arguments` in the future. I guess the easiest would be later since we already have some comprehensive docs.

Reply:
Good spot!

I added the extra param for now, though I'd note that these comments aren't exposed publically anywhere and so in general I'd be inclined to keep them simple as they are only for us devs who work on it anyway :)